initfiles/componentfiles/configxml/dfuserver.xsl

```diff
@@ -47,7 +47,7 @@
                <xsl:value-of select="@name"/>
             </xsl:attribute> 
 
-            <xsl:attribute name="daliservers"> <xsl:call-template name="getDaliServers">
+            <xsl:attribute name="daliServers"> <xsl:call-template name="getDaliServers">
```

Comment:
this looks sensible - was it deliberate?

Reply:
yes, it was.